src/de/tests.rs

```diff
@@ -699,7 +788,7 @@ fn test_leading_whitespace() {
     check_from_str_bytes_reader("  EmptyStruct1", Ok(EmptyStruct1));
 }
 
-fn check_from_str_bytes_reader<T: serde::de::DeserializeOwned + PartialEq + core::fmt::Debug>(
+pub fn check_from_str_bytes_reader<T: serde::de::DeserializeOwned + PartialEq + core::fmt::Debug>(
```

Comment:
Can you verify that this function cannot be reached in user-code?

Reply:
Oops, no need for this to be pub. Changed it back.